utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/hostapd_stations.lua

```diff
@@ -1,52 +1,78 @@
 local ubus = require "ubus"
 local bit32 = require "bit32"
 
+local function get_wifi_interfaces()
+  local conn = ubus.connect()
+  local ubuslist = conn:objects()
+  local interfaces = {}
+
+  for _,net in ipairs(ubuslist) do
+    if net.find(net,"hostapd.") then
+      local ifname = net:gsub("hostapd.", "")
+      table.insert(interfaces, ifname);
+    end
+  end
+  conn:close()
+  return interfaces;
+end
+
+local function is_ubus_interface(ubus_interfaces, interface)
+  for i=1,#ubus_interfaces do
+    if ubus_interfaces[i] == interface then return true end
+  end
+  return false
+end
+
 local function get_wifi_interface_labels()
   local u = ubus.connect()
   local status = u:call("network.wireless", "status", {})
```

Comment:
can't we remove this call ?

Reply:
no, because we want the encryption that is not contained in `hostapd_cli -i int status` :/

Reply:
:(